src/Uno.Templates/content/unoapp/MyExtensionsApp.1/MyExtensionsApp.1.csproj

```diff
@@ -85,6 +85,14 @@
       <!--#if (useHttp)-->
```

Comment:
This needs to change to `#if (useHttpBasic)`

The `useHttp` condition simply now means that http is enabled for any choice (basic, kiota, refit)